.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "0.2.0",
```

Comment:
.vscode/* files contain personal configurations. If you share your repository its likely that others do not setup their editors in the same way.

Add this file to git-ignored files and instead, add a .vscode/settings.json.template. This way, you suggest configuration to others instead of force it.

Reply:
You added `launch.json.template`. Cool 👍, but the original `launch.json` is still here. The purpose of all this is to not version `launch.json` and instead leave only the template file.